onboarding/src/Components/EmployeeProfile/ImplementationFormsToSend.js

```diff
@@ -48,6 +48,7 @@ function ImplementationFormsToSend(props) {
     return(
         <div className="card card-fluid">
             <div className="card-header">
+                <i class="bi bi-cloud-upload mr-2" style={{ fontSize: "18px" }}></i>
```

Comment:
`Warning: Invalid DOM property 'class'. Did you mean 'className'?`
`in i (created by ImplementationFormsToSend)`